kingpin/actors/group.py

```diff
@@ -89,11 +91,21 @@ def _build_actions(self):
         passed into this actors 'init_context' are also passed into the
         actors that we're intantiating.
         """
-        if not self.option('contexts'):
+        any_contexts = self.option('contexts') or self.option('context-file')
+        if not any_contexts:
             return self._build_action_group(self._init_context)
 
+        if self.option('contexts'):
+            context_data = self.option('contexts')
+            if self.option('context-file'):
+                raise exceptions.InvalidOptions(
```

Comment:
By re-using the word `contexts` above, you remove this complication. It clearly forces the user to do one or the other.